__tests__/cron/checkReferralReminder.ts

```diff
@@ -0,0 +1,116 @@
+import cron from '../../src/cron/checkReferralReminder';
+import { expectSuccessfulCron, saveFixtures } from '../helpers';
+import { DataSource } from 'typeorm';
+import createOrGetConnection from '../../src/db';
+import { Alerts, ALERTS_DEFAULT, User } from '../../src/entity';
+import { usersFixture } from '../fixture/user';
+import pino from 'pino';
+import { subMonths, subWeeks } from 'date-fns';
+
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+});
+
+beforeEach(async () => {
+  await saveFixtures(con, User, usersFixture);
+  jest.resetAllMocks();
+});
+
+const expectAffectedRows = async (count: number) => {
```

Comment:
I wouldn't recommend basing our tests on the logs. It's better to check the db directly

Reply:
I see. The part I needed this was to check whether something was not updated, which pulling the value from DB would be the same from start and when finished.

Reply:
So you can check that nothing changed if that's what you want

Reply:
Yep, we can do that. Just wanted something more explicit initially 😅